web-app/stories/Button.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './button.css';
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Button = ({ primary, backgroundColor, size, label, ...props }) => {
+  const mode = primary ? 'storybook-button--primary' : 'storybook-button--secondary';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/toost-net/toost-cloud/pullRequest?prid=8079923)
```suggestion
  const mode = primary ? "storybook-button--primary" : 'storybook-button--secondary';
```